database/migrations/sqls/20210326224122-capture-map-up.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE capture_map
+(
+    id uuid NOT NULL PRIMARY KEY,
+    image_url varchar NOT NULL,

```

Comment:
do we need the image urls for capture and field user?  would we every query on these?

Reply:
Don;t we need these if the web-map service wants to show the picture?